packages/xy-chart/src/series/CirclePackSeries.jsx

```diff
@@ -19,7 +21,18 @@ const defaultProps = {
 };
 
 function CirclePackSeries(props) {
-  return <PointSeries {...props} />;
+  const { xScale, data, yScale } = props;
+  const { data: modifiedData,
+          xScale: modifiedXScale,
```

Comment:
maybe I'm misinterpreting but I think you can get rid of this line since you aren't modifying the `xScale` ? also see my note on factoring out the `yScale` modifications, then all you'd have to return is the modified data (and it wouldn't have to be in an object either)
`const packedData = computeCirclePack(data, xScale);`

Reply:
I thought of modifying xScale at the first beginning but in the end it is not necessary. So I will remove xScale for now.